Tables/TicketsTable.sql

```diff
@@ -9,5 +9,6 @@ CREATE TABLE Tickets(
 	[Description] NVARCHAR(MAX),
 	EctsMark VARCHAR(1),
 	Grade INT,
-	Comment NVARCHAR(MAX)
+	Comment NVARCHAR(MAX),
+	Deadline DATE
```

Comment:
Should we add some default values for table columns?

Reply:
I don't think I got your point. Why would we do that?